golem_core/low/payment_event_collector.py

```diff
@@ -68,6 +71,7 @@ async def _get_event_resources(self, event: DebitNoteEvent) -> Tuple[DebitNote,
 
 class InvoiceEventCollector(PaymentEventCollector):
     @property
+    @api_call_wrapper()
```

Comment:
Also not here :)
In general, non-async functions never make api calls, as the api is async.

Reply:
Thanks for your review! I didn't notice it returns function straightaway, not the call. I'm removing the wrapper around properties then, it seems consequent calls of these properties are already wrapped with try-except blocks